single_kernel_mongo/managers/ldap.py

```diff
@@ -205,10 +205,14 @@ def get_ldap_connection_status(self) -> StatusBase:
         bind_dn = self.state.ldap.bind_user
         bind_password = self.state.ldap.bind_password
         base_dn = self.state.ldap.base_dn
+        cert_chain = self.state.ldap.chain
 
         if not base_dn:
             return BlockedStatus("Missing base DN.")
 
+        if not cert_chain:
+            return BlockedStatus("Missing chain.")
```

Comment:
```suggestion
            return BlockedStatus("Missing chain for ldap.")
```

Perhaps the needed user action can be added to the status


Also can you log something here